package day10;

public class Test {
	public static void main(String[] args) {
//		int i = 0;
//		try {//用try()来括住一段有可能出现异常的代码段
//			System.out.println(1);
//			System.out.println(1/i);
//			System.out.println(2);
//		}catch(Exception e) {
//			//当不知道捕获的是什么类型的异常时，可以直接使用所有异常的父类Exception
//			System.out.println(3);
//		}
//		finally {
//			System.out.println(4);
//		}
//		System.out.println("OK");
		
		String[] strs = new String[] {"a","b"};
	}
}

class A{
	int i;
}
